src/control/server/storage/bdev/backend_vmd.go

```diff
@@ -21,6 +21,9 @@ import (
 	"github.com/daos-stack/daos/src/control/server/storage"
 )
 
+// getVMD returns VMD endpoint address when provided string is a VMD backing device PCI address.
+// If the input string is not a VMD backing device PCI address, nil is returned and if not a valid
```

Comment:
This is not a good pattern, IMO. It would be better to return a sentinel error indicating that the supplied address string does not represent a VMD backing address. The caller can then decide how to handle it. This avoids spreading around responsibility for interpreting the result.

Reply:
- [ ] return sentinel Rather than nil for the case where the address string does not represent a VMD backing device